pywinauto/controls/atspi_controls.py

```diff
@@ -278,3 +278,37 @@ def select(self, start=0, end=None):
 
         # return this control so that actions can be chained.
         return self
+
+
+class ImageWrapper(atspiwrapper.AtspiWrapper):
+
+    """Wrap image controls"""
+
+    _control_types = ['Image', 'Icon']
+
+    # -----------------------------------------------------------
+    def __init__(self, elem):
+        """Initialize the control"""
+        super(ImageWrapper, self).__init__(elem)
+
+    def get_description(self):
+        """Get image description"""
+        self.element_info.image_get_description()
+
+    def get_locale(self):
+        """Get image locale"""
+        self.element_info.image_get_locale()
+
+    def get_size(self):
+        """Get image size.
+        Return a tupe with width and height"""
+        pnt = self.element_info.image_get_size()
+        return (pnt.x, pnt.y)
+
+    def get_bounding_box(self):
+        """Get image bounding box"""
+        return self.element_info.image_get_extents()
+
+    def get_position(self):
```

Comment:
Maybe `get_` is redundant in all method names. Not sure it's consistent with method names for other wrappers, but we have method `rectangle` instead of `get_rectangle` for example.

Reply:
Resolved with e358ab2